en/components/texthighlight.md

```diff
@@ -0,0 +1,340 @@
+---
+title: TextHighlight Directive - Native Angular | Ignite UI for Angular
+_description: The Ignite UI for Angular TextHighlight directive can be used to highlight parts of text and have an active highlight on one of them.
+_keywords: Ignite UI for Angular, UI controls, Angular widgets, web widgets, UI widgets, Angular, Native Angular Components Suite, Native Angular Components, Native Angular Controls, Native Angular Components Library, Angular TextHighlight Directive, IgxTextHighlight Directive
+---
+
+## TextHighlight Directive
+
+The `IgxTextHighlight` directive in Ignite UI for Angular is used to highlight parts of a text, providing options for case sensitive searches and to highlight only exact matches. It also allows the developer to keep an active highlight, which can be any of the already highlighted parts.
+
+### TextHighlight Demo
+<div class="sample-container loading" style="height: 260px; width: 600px;">
+    <iframe id="text-highlight-1" frameborder="0" seamless width="100%" height="100%" src="{environment:demosBaseUrl}/text-highlight-1" onload="onSampleIframeContentLoaded(this);"></iframe>
+</div>
+<div>
+    <button data-localize="stackblitz" class="stackblitz-btn" data-iframe-id="text-highlight-1" data-demos-base-url="{environment:demosBaseUrl}">view on stackblitz</button>
+</div>
+<div class="divider--half"></div>
+
+> [!NOTE]
+> To start using Ignite UI for Angular components in your own projects, make sure you have configured all necessary dependencies and have performed the proper setup of your project. You can learn how to do this in the [**installation**](https://www.infragistics.com/products/ignite-ui-angular/getting-started#installation) topic.
+
+### Usage
+
+To get started with the Ignite UI for Angular TextHighlight directive, let's first import the **IgxTextHighlightModule** in the **app.module.ts** file.
+
+```typescript
+// app.module.ts
+
+...
+import { IgxTextHighlightModule } from 'igniteui-angular';
+
+@NgModule({
+    ...
+    imports: [..., IgxTextHighlightModule],
+    ...
+})
+export class AppModule {}
+```
+
+Then, lets create a search box which we can use to highlight different parts of the text. We will use IgniteUI for Angular's InputGroup component in which we will add a text input with buttons for clear matches, find next and find previous and a button for specifying whether the search will be case sensitive or not. Also it has a label for how many matches we have found.
+
```

Comment:
1. It would be nice to reference the **IgxInputGroup** as a hyperlink to its respective page:
https://www.infragistics.com/products/ignite-ui-angular/angular/components/input_group.html
2. You can add an additional html code-snippet, which shows all the control modules we have imported for our application (for example: IgxButtonModule, IgxIconModule, IgxInputGroupModule, etc.)